packages/eb-deep-merge/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "rulesDirectory": [
+    "node_modules/tslint"
+  ],
+  "extends": [
+    "@empathybroker/eb-tslint"
+  ]
+}
```

Comment:
Remove unnecessary file.

Reply:
Can we rename the folder from `eb-deep-merge` to just `deep-merge`?

Reply:
Fix in the other PR.